landoapi/models/landing.py

```diff
@@ -128,3 +132,37 @@ class RevisionNotFoundException(Exception):
     def __init__(self, revision_id):
         super().__init__()
         self.revision_id = revision_id
+
+
+def _upload_patch_to_s3(patch, revision_id, diff_id):
+    """Save patch in S3 bucket.
+
+    Creates a temporary file and uploads it to an S3 bucket.
+    Requires PATCH_BUCKET_NAME to be provided as an environment variable.
+
+    Args:
+        patch: Text to be saved
+        revision_id: String ID of the revision (ex. 'D123')
+        diff_id: The integer ID of the raw diff
+
+    Returns
+        String representing the patch's URL in S3
+        (ex. 's3://{bucket_name}/D123_1.patch')
+    """
+    s3 = boto3.resource(
+        's3',
+        aws_access_key_id=os.getenv('AWS_ACCESS_KEY', None),
```

Comment:
Yes, if the environment variables aren't set, access and secret keys are set to None.
That will make it work in the way we discussed. I can add the comment that this variables are not set in production. Using `flask-boto3` would move this code to `app.py`. No other benefits.